Tests/test_DataStore_populate.py

```diff
@@ -0,0 +1,51 @@
+import unittest
+import os
+
+from unittest import TestCase
+from Store.DataStoreModule import DataStore
+
+FILE_PATH = os.path.dirname(__file__)
+TEST_DATA_PATH = os.path.join(FILE_PATH, "sample_data")
+
+
+class TestDataStorePopulate(TestCase):
+    def setUp(self):
+        self.sqlite = DataStore("", "", "", 0, ":memory:", db_type="sqlite")
+        self.sqlite.initialise()
+
+    def tearDown(self):
+        pass
+
+    def test_populate_reference(self):
+        """Test whether CSVs successfully imported to SQLite"""
+
+        # Check tables are created but empty
+        with self.sqlite.session_scope() as session:
+            nationalities = self.sqlite.getNationalities()
+            platform_types = self.sqlite.getPlatformTypes()
+
+        # There must be no entities at the beginning
+        self.assertEqual(len(nationalities), 0)
+        self.assertEqual(len(platform_types), 0)
+
+        # Import CSVs to the related tables
+        with self.sqlite.session_scope() as session:
+            self.sqlite.populateReference(TEST_DATA_PATH)
+
+        # Check tables filled with correct data
+        with self.sqlite.session_scope() as session:
+            nationalities = self.sqlite.getNationalities()
+            platform_types = self.sqlite.getPlatformTypes()
+            nationality_object = self.sqlite.searchNationality("UNITED KINGDOM")
+            platform_type_object = self.sqlite.searchPlatformType("TYPE-1")
+
+            # Check whether they are not empty anymore and filled with correct data
+            self.assertNotEqual(len(nationalities), 0)
```

Comment:
I'm ok with testing by _sampling_, but I think in this case it may be worth testing that all of the tables are non-empty.

This _may_ just catch us if a table-name gets changed, a CSV is wrongly named.

Hmm, maybe we don't need to.  Will the code already throw an error if a CSV file isn't found for a table?

Reply:
It just prints `method not found` when `addTo+table_name` isn't found. If the expected usage is importing CSV files for all tables, I can do the necessary changes.

Reply:
Aah, ok.  We're in `populate` here, not `initialise`.    That's fine - we're not meant to be checking all the tables are present, we're checking some have data. That's fine.